pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -84,6 +86,10 @@ type TriggerSpec struct {
 	// Subscriber is the addressable that receives events from the Broker that pass the Filter. It
 	// is required.
 	Subscriber duckv1.Destination `json:"subscriber"`
+
+	// Delivery configuration
```

Comment:
nit: can this comment include details similar to the original comment on the Broker delivery field?
```golang
	// Delivery is the delivery specification for Events within the Broker mesh.
	// This includes things like retries, DLQ, etc.
```

Reply:
The original comment is wrong and that's what this PR is trying to fix :smile: